### ISSUE TITLE
combat anim "on air"

### ISSUE THREAD

**Issue by GitMadKing** [Sphereserver/Source-X#688]
with COMBAT_SWING_NORANGE
the anim is performed at the end of the swing time (when ready =1)
but the hit is delivered only when "in range", close to the target.
anim should be performed also with the hit

**Comment by drk84** [comment]
yes it's not complete and probably already old

**Comment by cbnolok** [comment]
It "shouldn't" be performed also with the hit, that setting explicitly states that the swing (thus also the anim) happens regardlessly of the range. If you desire to play another anim, store the local.anim value on @hittry and play it again when you like

**Comment by GitMadKing** [comment]
yes, sorry, i meant saying that i want to play the anim ONLY at the hit, and not when swing ready. is there a way to avoid "auto"-anim when swing ready?

thank you

**Comment by cbnolok** [comment]
Yep, set local.anim to -1 in @hittry, or use different ini settings
